example/test/_features/step_definitions/steps.js

```diff
@@ -82,3 +82,8 @@ Then('I should see the {string} error', async function (content) {
   await this.page.waitForSelector('body', { timeout: 15000 });
   expect(await this.page.innerText('.validation-summary.error-summary')).to.include(content);
 }.bind(World));
+
+Then('I select field {string} and value {string}', async function (field, value) {
```

Comment:
On line 86 - I would remove the word, field and value before {string}, so that it is more consistent with the others, and reads a bit better above.

Reply:
It also reads a bit better if you have it the other way round. so ... async function (value, field)